index.html

```diff
@@ -1034,60 +1034,107 @@
 of an event that occurs nearly-simultaneously on both sites.
 
 
-## Personal Data {#data}
+## Data minimization {#data}
 
 <aside class="issue">
-  This section has not been edited for narrative.
-</aside>
-
-<aside class="issue">
-  There is not currently consensus among the authors and contributors to this
-  document about the below principles.
-  Please see <a href="https://github.com/w3ctag/privacy-principles/issues/150">this issue</a>
+  There is not currently consensus among the authors and contributors to this document about the below
+  principles. Please see <a href="https://github.com/w3ctag/privacy-principles/issues/150">this
+  issue</a>
   for more details.
 </aside>
 
-<div class="practice">
+<span class="practicelab">Sites, user agents and other parties should minimize the amount of data
+about people that is shared with or transferred between parties on the Web.</span>
 
-<p><span class="practicelab" id="user-agent-data-sharing">
-User agents should only share and present information about the user that
-the user can reasonably anticipate being shared, and which directly relate to the users overt, immediate goals.
-User agents and sites should be conservative when deciding if information is related to a users goals; the
-longer the chain of reasoning used to justify data collection, the less likely for that data
-collection to be ethical and principled.
-</span></p>
+Data minimization limits the risks of data being disclosed or misused and also makes it possible to
+more meaningfully provide people with understandable decisions about data about them.
 
-</div>
-
-Examples:
+<span class="practicelab">Web APIs should be designed to minimize the amount of data that is requested
```

Comment:
How much can we avoid passive voice here?

```suggestion
<span class="practicelab">Web APIs should be designed to minimize the amount of data that sites need to request to carry out their users' goals
```

would be a start ... "granularity" might be redundant since sites can only request a minimal amount of information if the grains of information are small.

Then ... do user controls belong in this principle? I could imagine saying that we should try to design APIs to maximize the amount of user control that UAs can provide, but that's not really about minimization anymore. More about #181 maybe.